airflow/models/baseoperator.py

```diff
@@ -1546,13 +1546,17 @@ def inherits_from_dummy_operator(self):
         return getattr(self, '_is_dummy', False)
 
 
-def chain(*tasks: Union[BaseOperator, "XComArg", Sequence[Union[BaseOperator, "XComArg"]]]):
+def chain(
+    *tasks: Union[
+        BaseOperator, "XComArg", EdgeModifier, Sequence[Union[BaseOperator, "XComArg", EdgeModifier]]
```

Comment:
At this point we should make this `BaseOperator, "XComArg", EdgeModifier` thing a type alias, maybe something like

```python
Chainable = Union[BaseOperator, "XComArg", EdgeModifier]
def chain(*tasks: Union[Chainable, Sequence[Chainable]]) -> None:
    ...
```

Reply:
Brilliant. I'm sure there will be other types that will be "chainable" in the future.  I agree the annotations are beginning to become unwieldy.